README.md

```diff
@@ -1,141 +1,11 @@
-# Mattapan Mapping
+# mattapanProject
```

Comment:
We should probably keep this file for future developers. We can update it as we move through the project with any process changes, etc.